### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by yefimchuk**

### CODE REVIEW

**Comment by IlyaKozhevnikov7** [comment]
Nice pull request man!!! Good work!!!

**Review by yefimchuk — COMMENTED** [review]
good

File: src/registration/registration.service.ts [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
+    constructor(@InjectModel(Registration.name) private regModel: Model<RegistrationDocument>) {
     }
 
+    async registrationUser(regDto: RegistrationUserDto): Promise<Registration> {
+
+        let {email, password} = regDto;
+        const isUsed = await this.regModel.findOne({email});
+        if (isUsed) {
+            throw new HttpException({
+                status: HttpStatus.BAD_REQUEST,
```
Comment (yefimchuk) [inline src/registration/registration.service.ts:21]: no bad request man

File: src/schemas/login.schema.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+
+export type LoginDocument = Login & Document;
+
+@Schema()
+export class Login {
+    @Prop()
+    email: string;
```
Comment (yefimchuk) [inline src/schemas/login.schema.ts:9]: good work

File: src/login/login.service.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+@Injectable()
+export class LoginService {
+
+    constructor(@InjectModel(Registration.name) private regModel: Model<RegistrationDocument>) {
+    }
+
+    async loginUser(regDto: LoginUserDto) {
+
+        let {email, password} = regDto;
+        const user = await this.regModel.findOne({email});
```
Comment (IlyaKozhevnikov7) [inline src/login/login.service.ts:19]: Man, cypher should find it? He knows exactly where it is!!!

**Comment by yefimchuk** [comment]
> Nice pull request man!!! Good work!!!

**Review by IlyaKozhevnikov7 — COMMENTED** [review]
Bad work man

File: src/login/login.service.ts (hunk 3 above)
Comment (yefimchuk) [reply src/login/login.service.ts:19]: molodec

**Pull request merged**